### ISSUE TITLE
Speed up ground-truth LoanStatus button

### ISSUE THREAD

**Issue by cdrini** [internetarchive/openlibrary#5917]
This button takes ~1-1.5s on every book page! We want to speed this up somehow.

### Proposal & Constraints
- Maybe request client-side when JS enabled? And use faster, bulk api for server-side.

**Comment by mekarpeles** [comment]
This plan is to load the Bulk Availability data initially (since get_sorted_editions already gives this to us). Furthermore, we have to change the `edition = page` assignment in the type/editions/view.html page so that the Bulk availability gets added to `page` which otherwise won't have it.

@mekarpeles has a diff on ol-dev1/ol-mek which has a working example

The final step is to async load the availability after page load.

TL;DR, this issue will be solved by #6065 (so closing).
